plugins/ep963x/fu-ep963x-device.c

```diff
@@ -47,8 +47,9 @@ fu_ep963x_device_write (FuEp963xDevice *self,
 }
 
 static gboolean
-fu_ep963x_device_write_icp (FuEp963xDevice *self,
-			    guint8 cmd, guint8 *buf, gsize bufsz,
+fu_ep963x_device_write_icp (FuEp963xDevice *self, guint8 cmd,
+			    const guint8 *buf, gsize bufsz,
+			    guint8 *bufout, gsize bufoutsz,
```

Comment:
I will admit this feels a bit hacky to pass the buf parameter twice, once in const and one not..  Would you consider instead excluding the qualifier check for the compiler in this function?

Reply:
The right thing to do is to not pass in the same buf. I'll fix.

Reply:
Done.